Avatar.css

```diff
@@ -0,0 +1,115 @@
+@import "Css/main.css";
+:root{
+    /* Colors  */
+     --primary-color:#420039;
+     --secondary-color:#E1DEE9 ;
+     --plain-primary:white;
+     --danger:red;
+     --success:green;
+ 
+    /* Typography */
+    --designed-font: 'Kaushan Script', cursive;
+ 
+ }
+ 
+ 
+
+
+.component-heading{
+    font-size: 3rem;
+    /* padding:.5rem 1rem; */
+    margin:0;
+}
+
+
+
+.avatar-img{
+    width:80px;
+    height:80px;
```

Comment:
You have used px unit mostly, I would suggest you to rem instead.

Reply:
> You have used px unit mostly, I would suggest you to rem instead.

Thanks sararn I will work on it.